### PR TITLE
front/login-page

### PR DESCRIPTION
**Pull request by AdamProgramista**
Check this my friends :)

### CODE REVIEW

File: front/src/components/Button/Button.css [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  font-size: 2rem;
+  background: #2980b9;
+  padding: 0.5rem 2rem;
+  margin: 0 0.5rem;
+  cursor: pointer;
+}
+
+.btn:hover {
+  background: #1a5073;
+}
```
Comment (monikasudol) [inline front/src/components/Button/Button.css]: please add empty lines at the end of every file

File: front/src/components/Input/Input.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
+import './Input.css';
+
+class Input extends Component {
+  render() {
+    return (
+      <div>
+        <input className='input' placeholder={this.props.text}/>
```
Comment (monikasudol) [inline front/src/components/Input/Input.js]: please change className

File: front/src/components/Input/Input.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+      </div>
+    );
+  }
+}
+
+export default Input;
```
Comment (monikasudol) [inline front/src/components/Input/Input.js]: same like above

File: front/src/components/Input/Input.js (hunk 2 above)
Comment (AdamProgramista) [reply front/src/components/Input/Input.js]: Why? It's ok.

Comment (monikasudol) [reply front/src/components/Input/Input.js]: we should discuss about this issue

File: front/src/components/Button/Button.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
+import './Button.css';
+
+class Button extends Component {
```
Comment (yuriyyakym) [inline front/src/components/Button/Button.js]: It could be a functional component

File: front/src/components/Input/Input.css [hunk 5]
```diff
@@ -0,0 +1,9 @@
+.input {
+  width: 40rem;
+  font-size: 3rem;
+  background-color: rgba(255, 255, 255, 0.5);
+  padding: 0.5rem 1rem;
+  border-radius: 0.25rem;
+}
+
```
Comment (yuriyyakym) [inline front/src/components/Input/Input.css]: redundant empty line

**Comment by monikasudol** [comment]
please hurry up my Friend!

**Pull request merged**